provisioner.go

```diff
@@ -185,8 +185,13 @@ func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher
 
 // provisionPackage provisions a package
 func (p *defaultProvisioner) provisionPackage(ctx context.Context, d Dispatcher, paths Paths, s *ProvisionStatusPackage, m mover, name, version, pathUnzipSrc, pathDirectory string, finish func() error) (err error) {
+	_, err = os.Stat(pathDirectory)
```

Comment:
@lunny Their disembed funcs use the same signature and they don't conflict in any way. Since go-bindata can't download files, I wrote my own repo from scratch.